docs/synapse/userguides/model_updates/update_v2_195_0.rst

```diff
@@ -0,0 +1,729 @@
+
+
+  ``vehicle``
+    The air craft which traveled the flight.
+
+
+``transport:land:vehicle``
+  The form had the following properties added to it:
+
+
+  ``desc``
+    A description of the vehicle.
+
+
+  ``manufacturer``
+    The organization which manufactured the vehicle.
+
+
+  ``manufacturer:name``
+    The name of the organization which manufactured the vehicle.
+
+
+  ``max:cargo:mass``
+    The maximum mass the vehicle can carry as cargo.
+
+
+  ``max:cargo:volume``
+    The maximum volume the vehicle can carry as cargo.
+
+
+  ``max:occupants``
+    The maximum number of occupants the vehicle can hold.
+
+
+  ``operator``
+    The contact information of operator of the vehicle.
+
+
+  ``phys:height``
+    The height of the vehicle.
+
+
+  ``phys:length``
+    The length of the vehicle.
+
+
+  ``phys:mass``
+    The mass of the vehicle.
+
+
+  ``phys:volume``
+    The cubed volume of the vehicle.
+
+
+  ``phys:width``
+    The width of the vehicle.
+
+
+  ``type``
+    The type of land vehicle.
+
+
+``transport:sea:vessel``
+  The form had the following properties added to it:
+
+
+  ``manufacturer``
+    The organization which manufactured the vessel.
+
+
+  ``manufacturer:name``
+    The name of the organization which manufactured the vessel.
+
+
+  ``max:cargo:mass``
+    The maximum mass the vessel can carry as cargo.
+
+
+  ``max:cargo:volume``
+    The maximum volume the vessel can carry as cargo.
+
+
+  ``max:occupants``
+    The maximum number of occupants the vessel can hold.
+
+
+  ``owner``
+    The contact information of the owner of the vessel.
+
+
+  ``phys:height``
+    The height of the vessel.
+
+
+  ``phys:length``
+    The length of the vessel.
+
+
+  ``phys:mass``
+    The mass of the vessel.
+
+
+  ``phys:volume``
+    The cubed volume of the vessel.
+
+
+  ``phys:width``
+    The width of the vessel.
+
+
+  ``serial``
+    The manufacturer assigned serial number of the vessel.
+
+
+  ``type``
+    The type of vessel.
+
+
+
+*************
+Updated Types
+*************
+
+``geo:telem``
+  The type interface has been modified to inherit from the ``phys:object``
+  and ``geo:locatable`` interfaces.
+
+
+``it:host``
+  The type interface has been modified from ``inet:service:object'`` to
+  inherit from the ``inet:service:object`` and ``phys:object`` interfaces.
+
+
+``mat:item``
+  The type interface has been modified to inherit from the ``phys:object``
+  and ``geo:locatable`` interfaces.
+
+
+``ps:vitals``
+  The type interface has been modified to inherit from the ``phys:object``
+  interface.
+
+
+``transport:air:craft``
+  The type interface has been modified to inherit from the ``transport:vehicle``
+  interface.
+
+
+``transport:air:flight``
+  The type interface has been modified to inherit from the ``transport:trip``
+  interface.
+
+
+``transport:land:vehicle``
+  The type interface has been modified to inherit from the ``transport:vehicle``
+  interface.
+
+``transport:sea:vessel``
```

Comment:
I don't think this actually renders differently but for some reason there's only one blank line between these rather than two?